airgun/widgets.py

```diff
@@ -635,12 +635,13 @@ class Search(Widget):
     search_field = TextInput(
         locator=(
             ".//input[@id='search' or contains(@placeholder, 'Filter') or "
-            "@ng-model='table.searchTerm' or contains(@ng-model, 'Filter')]"
+            "contains(@ng-model, 'Filter') or @data-autocomplete-id='searchBar' "
```

Comment:
No need to remove `ng-model="table.searchTerm"` as I can see that is still valid in the content hosts search

Reply:
updated the pr.